platforms/ios/src/TangramMap/TGMarker.mm

```diff
@@ -57,13 +57,24 @@ - (instancetype)initWithMapView:(TGMapViewController *)mapView
     return self;
 }
 
-- (void)setStyling:(NSString *)styling
+- (void)setStylingString:(NSString *)styling
```

Comment:
We should have api name consistency between iOS and Android:
Android: `setStylingFromString` and `setStylingFromPath`
iOS: `setStylingString` and `setStylingPath`